src/renderer/globals.scss

```diff
@@ -190,3 +234,33 @@ a.is-button {
 .modal-card {
   z-index: 200;
 }
+
+.button .is-rounded .is-primary {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-primary {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-primary[disabled] {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-rounded.is-cancel {
+  color: $brand-primary !important;
+  background-color: transparent !important;
+  font-weight: 500 !important;
+  border-color: $brand-primary !important;
+}
+
+.button {
```

Comment:
Can we move this to be the first of all the button configuration?

```
.button { ... }
.button .xx { ... }
.button .xx .yy { ... }
```

Reply:
fixed